driver/gl/raster_cache.go

```diff
@@ -0,0 +1,66 @@
+package gl
```

Comment:
I would rename this to `svg_cache.go` for clarity

Reply:
agreed